tests/resample/conftest.py

```diff
@@ -11,3 +21,71 @@ def wcs_gwcs():
     shape = (1000, 1000)
     pscale = 0.06 / 3600
     return make_gwcs(crpix, crval, pscale, shape)
+
+
+@pytest.fixture(scope="module")
+def nrcb5_wcs_wcsinfo():
+    """ Returns both wcs and wcsinfo """
+    path = Path(__file__).parent.parent / DATADIR / "nrcb5-wcs.asdf"
+    with asdf.open(path, lazy_load=False) as asdf_file:
+        wcs = asdf_file.tree["wcs"]
+        wcs.bounding_box = wcs_bbox_from_shape(wcs.array_shape)
+        wcsinfo = asdf_file.tree["wcsinfo"]
+        return wcs, wcsinfo
+
+
+@pytest.fixture(scope="module")
+def nrcb5_many_fluxes(nrcb5_wcs_wcsinfo):
+    np.random.seed(0)
+
+    exptime = 3.1415 + 100.0 * np.random.random()
+    model = make_nrcb5_model(nrcb5_wcs_wcsinfo, exptime=exptime)
+    model["dq"][:, :] = 1
+
+    patch_size = 21
+    patch_area = patch_size * patch_size
+    p2 = patch_size // 2
+    # add border so that resampled partial pixels can be isolated
+    # in the segmentation:
+    border = 4
+    pwb = patch_size + border
+
+    fwhm2sigma = 2.0 * math.sqrt(2.0 * math.log(2.0))
```

Comment:
Just use this since `astropy` is already dependency?

https://github.com/astropy/astropy/blob/ea320a519eae450ec542f8009ac8d4a5a728a24f/astropy/stats/funcs.py#L58

Reply:
@pllim How do you know all of this?

Reply:
Maybe deprecate it in favor of all caps?

Reply:
I'll open an issue